src/components/freshness-bar/FreshnessBar.tsx

```diff
@@ -0,0 +1,46 @@
+import { differenceInMilliseconds, format } from 'date-fns';
+import React from 'react';
+import * as R from 'ramda';
+
+import Tooltip from '../tooltip';
+import { Bar } from './Bar';
+
+type BarSizes = 'sm' | 'md';
+
+export interface Props {
+  /** expiration date given as `Date`(e.g. via new Date(model.createdAt)) */
+  expirationDate: Date;
```

Comment:
now the old `time` prop is replaced with `expirationDate`, and it receives just `Date`
all of the usages in `hopsworks` were like `<FreshnessBar time={data.created.replace('T', ' ')} />`
now it is used it like `<FreshnessBar expirationDate={data.created} />`

previously component would manually remove `T` inside ISO-formatted string and do finicky stuff
just removed that crap and inserted a date picker in a storybook
![image](https://user-images.githubusercontent.com/19791699/184220737-6d767823-d006-4ef5-a235-8244c97c275f.png)